code/modules/trade/datums/trade_stations_presets/nt_cruisers.dm

```diff
@@ -0,0 +1,55 @@
+/datum/trade_station/nt_cruisers
+    icon_states = "nt_cruiser" // Because we lack NT noncombat ships.
+    spawn_cost = 2
+    spawn_probability = 15
+    markup = 0.6
+    name_pool = list(
+        "NTV 'Faith'" = "'NeoTheology Vessel 'Faith' something, something, NeoTheology is good, buy us, convert, something something",
+        "NTV 'Hope'" = "'NeoTheology Vessel 'Hope' something something, this PR is unfinished and I need feedback something, something."
+    )
+
+    //Types of items sold by the station
+    assortiment = list(
+        "Biomatter Products?" = list(
+            /obj/item/weapon/storage/pouch/small_generic,
+            /obj/item/weapon/storage/pouch/medium_generic,
+            /obj/item/weapon/storage/pouch/large_generic,
+            /obj/item/weapon/storage/pouch/medical_supply,
+            /obj/item/weapon/storage/pouch/engineering_tools,
+            /obj/item/weapon/storage/pouch/engineering_supply,
+            /obj/item/weapon/storage/pouch/tubular,
+            /obj/item/weapon/storage/pouch/tubular/vial,
+            /obj/item/weapon/storage/pouch/ammo,
+            /obj/item/weapon/storage/pouch/medical_supply,
+            /obj/item/clothing/accessory/holster,
+            /obj/item/clothing/accessory/holster/armpit,
+            /obj/item/clothing/accessory/holster/waist,
+            /obj/item/clothing/accessory/holster/hip,
+        ),
+        "Energy" = list(
+            /obj/item/weapon/gun/energy/taser,
+            /obj/item/weapon/gun/energy/nt_svalinn,
+            /obj/item/weapon/gun/energy/laser,
+            /obj/item/weapon/gun/energy/ionrifle,
+            /obj/item/weapon/gun/energy/plasma,
+            /obj/item/weapon/gun/energy/plasma/destroyer
+// Probably not a good idea.
```

Comment:
I think these are fine if they have an amount range of (0,1) and the price tag to match them. Same concept to all OP guns that we can add.